oeq-ts-rest-api/test/Search.test.ts

```diff
@@ -132,3 +132,17 @@ describe('Search for attachments', () => {
     }
   );
 });
+
+describe('Build URL for downloading search result', function () {
```

Comment:
Should this perhaps be:

```suggestion
describe('Exports search results for the specified search params', function () {
```

And then perhaps add another `it` which tests calling the generated URL?

Reply:
Good idea! A new test has been added.